### ISSUE TITLE
Global SCSS files failing to import with new version of carbon-components

### ISSUE THREAD

**Issue by bryanvpham** [carbon-design-system/carbon#6367]
- [x] `carbon-components`
- [x] `carbon-components-react`

## Detailed description

I updated the two  eps mentioned above to **10.14.0** so that I could get bug fixes. But now SCSS imports of certain `carbon-components files` don't work anymore. It was working fine on old versions. I'm importing the same way as the carbon react tutorial teaches.

I tried deleting node_modules and did `npm cache clean --force`.

> Is this issue related to a specific component?

`carbon-components`

> What did you expect to happen? What happened instead? What would you like to
> see changed?

That things would work normally

> What browser are you working in?

Chrome latest.

> What version of the Carbon Design System are you using?

**10.14.0**

## Steps to reproduce the issue

1. Install latest carbon components.
2. Do the imports based on carbon.tutorial part 2.

### Carbon tutorial

https://www.carbondesignsystem.com/tutorial/react/step-2#style-landing-page

### Tree structure

![image](https://user-images.githubusercontent.com/16672406/86187675-5c403400-bb0a-11ea-8aa9-cff58ccf47df.png)

### Error message
```
./src/Containers/App/_app-container.scss (./node_modules/css-loader/dist/cjs.js??ref--5-oneOf-3-1!./node_modules/postcss-loader/src??postcss!./node_modules/sass-loader/dist/cjs.js??ref--5-oneOf-3-3!./src/Containers/App/_app-container.scss)
SassError: File to import not found or unreadable: carbon-components/scss/globals/scss/vendor/@carbon/type/scss/font-family.scss.
        on line 1 of /Users/bryan/Work/src/tdm/src/webapp/src/Containers/App/_app-container.scss
>> @import 'carbon-components/scss/globals/scss/vendor/@carbon/type/scss/font-f
   ^

```

**Comment by joshblack** [comment]
Hey there @bryanvpham! 👋  Not sure what exactly happened with the project to cause it to not work, but I believe this file still exists between versions. Using unpkg, we can see it available in 10.13.0 and 10.14.0 under the same name/location:

- [10.13](https://unpkg.com/browse/carbon-components@10.13.0/scss/globals/scss/vendor/@carbon/type/scss/_font-family.scss)
- [10.14](https://unpkg.com/browse/carbon-components@10.14.0/scss/globals/scss/vendor/@carbon/type/scss/_font-family.scss)

**Comment by bryanvpham** [comment]
Not sure what happened either but cleaning up and installing again and the files are there. I will close this issue.
